Soomla/Assets/Plugins/Soomla/Profile/SoomlaProfile.cs

```diff
@@ -31,6 +31,9 @@ namespace Soomla.Profile
 	public class SoomlaProfile
 	{
 		static SoomlaProfile _instance = null;
+
+		internal static bool nativeModulesInitialized = false;
```

Comment:
Flag for receiving info that native part was initialized. If we don't process native initialization, `OnProfileInitialized` will be invoked twice.